app.config.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
     name: 'Skenovacia aplikácia',
     slug: 'enforcement-scan-app',
     scheme: 'enforcement-scan-app',
-    version: '1.0.2',
```

Comment:
If it's bugfix, we bump last number, if it's new feature or a change (not a bugfix), we bump middle number.

Don't be afraid to bump middle number :)